### DISCUSSION TITLE
Dispenser and Dropper side top and bottom indicators

### DISCUSSION THREAD

**Discussion by waffleeiron** [BedrockTweaks/Files#331] [Ideas]
Since directional indicators are impossible due to Mojang's laziness how about only indicators showing which of those two blocks you have when looking from side, top or bottom. Like square indicators on top, bottom and side of droppers, and respectively circular indicators for dispensers?

**Comment by PAPISOP** [comment]
Not possible 
Read this to understand https://becomtweaks.github.io/docs/misc/impossible-packs.html#_5-directional-droppers-and-dispensers

**Reply by waffleeiron** [reply]
I didn't mean like making indicators for the direction they are facing, only indicators that display which block is present. I got it to work myself, I can give you my pack and you'll see what I mean

**Reply by PAPISOP** [reply]
I understood then seems to be a valid suggestion

**Reply by NSPC911** [reply]
> I didn't mean like making indicators for the direction they are facing, only indicators that display which block is present. I got it to work myself, I can give you my pack and you'll see what I mean

I believe [Clear Vanilla](https://mcpedl.com/clear-wire-texture-pack/) has the indicators, but I'm not sure if DrAV will allow it being added.
